### PR TITLE
Monitoring apps logs 1.0.0

### PR DESCRIPTION
**Pull request by imranMnts**
TagCheck

### What to do if the build fails?

At the moment contributors do not have access to the CI workflow triggered by StepLib PRs. In case of a failed build, we ask for your patience. Maintainers of Bitrise Steplib will sort it out for you or inform you if any further action is needed.

### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [x] __I will not move an already shared step version's tag to another commit__
- [x] I read the [Step Development Guideline](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md)
- [x] I have a test for my Step, which can be run with `bitrise run test` (in the step's repository)
- [x] I did run `bitrise run audit-this-step` (in the step's repository - note, if you don't have this workflow in your `bitrise.yml`, [you can copy it from the step template](https://github.com/bitrise-steplib/step-template/blob/master/bitrise.yml).)
- [x] I read and accept the [Abandoned Step policy](https://github.com/bitrise-io/bitrise-steplib#abandoned-step-policy)

**New Step**
Thank you for the new Step share! The CI check might will fail due to our extended validation engine. Nothing to worry about yet, we will get back to you shortly.

### CODE REVIEW

File: steps/monitoring-apps-logs/1.0.0/step.yml [hunk 1]
```diff
@@ -0,0 +1,48 @@
+title: Mobile apps logs
+summary: |
+  Check if your Android app kept any logs in your release build
+description: "Check if an Android application contain any logs in the release app\nIf
+  yes, this step will fail and generate a fail where you can see the count \n\n###
```
Comment (koral--) [inline steps/monitoring-apps-logs/1.0.0/step.yml:5]:
```suggestion
  yes, this step will fail and generate a failure where you can see the count \n\n###
```

File: steps/monitoring-apps-logs/1.0.0/step.yml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+published_at: 2021-03-24T14:10:22.874866+01:00
+source:
+  git: https://github.com/imranMnts/bitrise-step-monitoring-apps-logs.git
+  commit: 0791e5e935687943fcda2a3766621d0270fec15c
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+project_type_tags:
+- android
+- react-native
```
Comment (koral--) [inline steps/monitoring-apps-logs/1.0.0/step.yml:20]: Why not eg. ionic?

File: steps/monitoring-apps-logs/1.0.0/step.yml [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+is_always_run: true
+is_skippable: false
+inputs:
+- check_android: "yes"
+  opts:
+    description: To check or not android permissions
+    is_expand: true
+    is_required: true
+    summary: Do you want check android permissions?
+    title: Check Android
```
Comment (koral--) [inline steps/monitoring-apps-logs/1.0.0/step.yml:37]: It should be rather called "Check permissions"

Comment (imranMnts) [reply steps/monitoring-apps-logs/1.0.0/step.yml:37]: I updated that, thanks

**Comment by imranMnts** [comment]
Moved to #2968

**Pull request closed without merging**